releases/.gitignore

```diff
@@ -0,0 +1 @@
+!/bins
```

Comment:
Somehow this doesn't work for me? I'm getting

```
$ git status 
HEAD detached at pr/4339/head
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   v0.2.2-x86_64-linux.SHA256SUMS.asc

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	bins/

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
My `git status` has the bins under `releases/bins`, which is still showing up untracked in git

Reply:
https://github.com/fedimint/fedimint/pull/4367